tests/Writing/CombinedManipulationsTest.php

```diff
@@ -618,6 +618,56 @@ public function testRemoveOtherSessionRefreshKeepChanges()
         $this->assertEquals('Value', $node->getPropertyValue('newprop'));
     }
 
+    /**
+     * It should throw an exception when accessing a property of a node created
+     * and persisted earlier in the session, but which has since been deleted
+     * by a different session. (refresh should not throw an exception).
+     */
+    public function testRemoveNewNodeInOtherSessionDiscardChanges()
+    {
+        $this->setExpectedException(
+            'PHPCR\PathNotFoundException', 
+            'Property bar'
```

Comment:
yep, lets not force the message. and only put the expectation right before the line that should trigger the exception